src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -684,36 +653,71 @@ private static ISymbol GetEventAccessor(IEventSymbol iEvent, IOperation operatio
 
                 if (accessor != null)
                 {
-                    CheckOperationAttributes(operation, context, platformSpecificOperations, platformSpecificMembers, msBuildPlatforms, accessor);
+                    CheckOperationAttributes(operation, context, platformSpecificOperations, platformSpecificMembers, msBuildPlatforms, accessor, false);
                 }
-                else
+            }
+            else if (symbol is IMethodSymbol method)
+            {
+                AnalyzeTypeParameters(operation, context, platformSpecificOperations, platformSpecificMembers, msBuildPlatforms, method);
+            }
+
+            static void CheckTypeArguments(ImmutableArray<ITypeSymbol> namedTypes, IOperation operation, OperationAnalysisContext context,
```

Comment:
I think the parameter should be renamed as type arguments are not guaranteed to be named type, then can be any type symbol:
```suggestion
            static void CheckTypeArguments(ImmutableArray<ITypeSymbol> typeArguments, IOperation operation, OperationAnalysisContext context,
```